src/bin/pgcopydb/cli_clone_follow.c

```diff
@@ -65,6 +65,7 @@
 	"  --create-slot                 Create the replication slot\n" \
 	"  --origin                      Use this Postgres replication origin node name\n" \
 	"  --endpos                      Stop replaying changes when reaching this LSN\n" \
+	"  --use-copy-binary			 Use the COPY BINARY format for COPY operations\n" \
```

Comment:
```suggestion
	"  --use-copy-binary             Use the COPY BINARY format for COPY operations\n" \
```

Please update documentation files as well.